localization/ndt_scan_matcher/README.md

```diff
@@ -257,3 +259,21 @@ This is a function that uses de-grounded LiDAR scan to estimate the scan matchin
 | ------------------------------------- | ------ | ------------------------------------------------------------------------------------- |
 | `estimate_scores_for_degrounded_scan` | bool   | Flag for using scan matching score based on de-grounded LiDAR scan (FALSE by default) |
 | `z_margin_for_ground_removal`         | double | Z-value margin for removal ground points                                              |
+
+## 2D rea-time covariance estimation
```

Comment:
If this function consumes much calculation resource as that may spoil healthy system behavior, could you inform that in this README ?